app/mailers/observer_mailer.rb

```diff
@@ -1,28 +1,43 @@
 class ObserverMailer < ApplicationMailer
-  def on_observer_added(observation, reason)
+  def observer_added_confirmation(observation, reason)
```

Comment:
NAMING alert: I've been naming / re-naming these emails kind of willy nilly but one emerging pattern is that emails that are triggered by a user and go to that same user (eg: You cancelled your own request!) are considered `confirmation`s while actions triggered by another user are called `notification`s. If we think that is a good pattern, I think both of these should be `notification`s. Thoughts? 

There is another school of thought that those are both redundant words in the context of an email and should just be removed from all email names.